django_react_proj/users-fe/src/components/NewUserForm/NewUserForm.js

```diff
@@ -36,30 +38,44 @@ const NewUserForm = ({ user, resetState, toggle }) => {
       name,
       password,
       deadline,
+      deadlinev2,
     });
     resetState();
     toggle();
   };
   return (
     <Form onSubmit={user ? editUser : createUser}>
       <FormGroup>
-        <Label for="name">Nazwa zadania:</Label>
+        <Label for="name">Task name:</Label>
         <Input type="text" name="name" onChange={onNameChange} value={defaultIfEmpty(name)} />
       </FormGroup>
       <FormGroup>
-        <Label for="deadline">Data wykonania:</Label>
+        <Label for="deadline">Time interval:</Label>
         <FormGroup>
-          <TimePicker
-            format="H:m"
-            disableClock="true"
-            name="deadline"
-            onChange={setDeadline}
-            value={defaultIfEmpty(deadline)}
-          />
+          <Label>Od:</Label>
+          <FormGroup>
+            <TimePicker
+              format="H:m"
+              disableClock="true"
+              name="deadline"
+              onChange={setDeadline}
+              value={defaultIfEmpty(deadline)}
+            />
+          </FormGroup>
+          <Label>Do:</Label>
+          <FormGroup>
+            <TimePicker
+              format="H:m"
+              disableClock="true"
+              name="deadlinev2"
+              onChange={setDeadlinev2}
+              value={defaultIfEmpty(deadline)}
+            />
+          </FormGroup>
         </FormGroup>
       </FormGroup>
       <FormGroup>
-        <Label for="password">Opis:</Label>
+        <Label for="password">Description:</Label>
```

Comment:
You call it `Description` but implementation says it's password

Reply:
I will changes all names of variables in db and react in a matter of days.